tabular/src/autogluon/tabular/models/tab_transformer/TabTransformer.py

```diff
@@ -0,0 +1,160 @@
+#Code in this script based on work by Milan Cvitkovic, 
```

Comment:
Why is this file (and others) capitalized?

Reply:
Legacy, from the scientists. No problem with changing this to tab_transformer.py.

Reply:
Lets do that for consistency

Reply:
Changed file names to snake case.